app/src/utils/formatter.ts

```diff
@@ -0,0 +1,9 @@
+export const reformatDate = (date: string) => {
+  if (date) {
+    const dateArray = date.split("");
+    const day = Number(dateArray.slice(0, 2).join(""));
+    const month = dateArray.slice(3, 5).join("");
+    const year = Number(dateArray.slice(6, 10).join(""));
```

Comment:
There is no need to do a split.  Use a .substring

Reply:
This method could be done by using moment to parse the date.  Then a .toString with the specific format required.